public/app/features/alerting/unified/components/rules/EditRuleGroupModal.tsx

```diff
@@ -176,11 +179,21 @@ export function EditCloudGroupModal(props: ModalProps): React.ReactElement {
   const { namespace, group, onClose, intervalEditOnly, folderUid } = props;
 
   const styles = useStyles2(getStyles);
-  const dispatch = useDispatch();
-  const { loading, error, dispatched } =
-    useUnifiedAlertingSelector((state) => state.updateLotexNamespaceAndGroup) ?? initialAsyncRequestState;
   const notifyApp = useAppNotification();
 
+  /**
+   * This modal can take 3 different actions, depending on what fields were updated.
+   *
+   *  1. update the rule group details without renaming either the namespace or group
+   *  2. rename the rule group, but keeping it in the same namespace
+   *  3. move the rule group to a new namespace, optionally with a different group name
```

Comment:
how is that possible from this modal? I don't see the way to move the group to a different namespace 🤔

Reply:
ok, now I see your comments about having the possibility in the future 😎